docs/user-documentation/getting-started-with-islandora-claw.md

```diff
@@ -0,0 +1,57 @@
+# Getting Started with Islandora CLAW
+
+## How to Add a User
+1. Log in under admin credentials.
+2. Click the **Manage** tab from the top toolbar.
+3. Select **People** tab from the resulting menu. 
+ ![Alt text](../assets/people_page.png "People page")
```

Comment:
Good question This is again what the hack/doc document had. I started making some new screenshots, but I wanted to get this feedback first. @kimpham54 this morning already made a PR on my CLAW fork with additional screenshots I will soon include in this PR.